pkg/plugins/generic/generic_plugin.go

```diff
@@ -195,15 +195,18 @@ func needDrainNode(desired sriovnetworkv1.Interfaces, current sriovnetworkv1.Int
 		configured := false
 		for _, iface := range desired {
 			if iface.PciAddress == ifaceStatus.PciAddress {
-				// TODO: no need to perform further checks if ifaceStatus.NumVfs equals to 0
-				// once https://github.com/kubernetes/kubernetes/issues/109595 will be fixed
+				if ifaceStatus.NumVfs == 0 {
```

Comment:
nit: can you add the check after `configured` ?

Reply:
done